ThinkSharp.Licensing/Licensing/HardwareIdentifier.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) Jan-Niklas Schäfer. All rights reserved.  
 // Licensed under the MIT License. See LICENSE file in the project root for full license information.
+
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
 
-namespace ThinkSharp.Licensing
+namespace ThinkSharp.Licensing.Licensing
```

Comment:
When using "ThinkSharp.Licensing" as root namespace, it is consequent to have "Licensing.Licensing" for files within the Licensing folder.

However, I dont like to have the same name twice in the namespace. What do you think about moving the files within the 'Licensing' folder up to the root folder (ThinkSharp.Licensing next to the project file)?

Reply:
I also don't like the "Licensing.Licensing". I have the feeling, that the root namespace should contain (only) the classes that the users need for the "standard"-use-case. I will look up what that is and propose something.

However, maybe we should do this in a second pull request. Merging namespace reorganization with functional changes makes it hard to see the real changes afterwards.